.github/workflows/release-check.yml

```diff
@@ -0,0 +1,62 @@
+name: Release Checker
+
+on:
+  pull_request_target:
+    paths: [
+      "dispatch_examples/greeter/Cargo.toml",
+      "frc42_dispatch/Cargo.toml",
+      "frc42_dispatch/hasher/Cargo.toml",
+      "frc42_dispatch/macros/Cargo.toml",
+      "frc42_dispatch/macros/example/Cargo.toml",
+      "frc46_token/Cargo.toml",
+      "frc53_nft/Cargo.toml",
+      "fvm_actor_utils/Cargo.toml",
+      "fvm_dispatch_tools/Cargo.toml",
+      "testing/integration/Cargo.toml",
+      "testing/test_actors/Cargo.toml",
+      "testing/test_actors/actors/basic_nft_actor/Cargo.toml",
+      "testing/test_actors/actors/basic_receiving_actor/Cargo.toml",
+      "testing/test_actors/actors/basic_token_actor/Cargo.toml",
+      "testing/test_actors/actors/basic_transfer_actor/Cargo.toml",
+      "testing/test_actors/actors/frc46_factory_actor/Cargo.toml",
+      "testing/test_actors/actors/frc46_test_actor/Cargo.toml",
+      "testing/test_actors/actors/frc53_test_actor/Cargo.toml",
+      "testing/test_actors/actors/frc46_factory_token/token_impl/Cargo.toml"
```

Comment:
Unfortunately, that isn't that straighforward.

Reply:
Ack - bummer.  I guess this in part because GitHub Actions doesn't support YML anchors?
Maybe for now just add a comment that any changes here should also be made in releaser.yml (and vice versa)